src/components/Details.js

```diff
@@ -0,0 +1,74 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { fetchCompanyDetails } from '../redux/companiesDetails/companiesDetailsSlice';
+import './Details.css';
+
+const Details = () => {
+  const company = useSelector((state) => state.companiesDetails.company);
+  const dispatch = useDispatch();
+  const { smbl } = useParams();
+  useEffect(() => {
+    dispatch(fetchCompanyDetails(smbl));
+  }, [dispatch]);
+
+  return (
+    <div className="Details">
+      <div className="Details-card-content">
+        <img
+          src={company?.image}
+          alt="company logo"
+          className="Details-company-logo"
+        />
+        <p>
+          (
+          {company?.symbol}
+          )
+        </p>
+        <h2>{company?.companyName}</h2>
+        <table>
+          <tbody>
+            <tr>
+              <th scope="col">CEO</th>
+              <td>{company?.ceo}</td>
+            </tr>
+            <tr>
+              <th scope="col">Sector</th>
+              <td>{company?.sector}</td>
+            </tr>
+            <tr>
+              <th scope="col">Price</th>
+              <td>{company?.price}</td>
+            </tr>
+            <tr>
+              <th scope="col">Currency</th>
+              <td>{company?.currency}</td>
+            </tr>
+            <tr>
+              <th scope="col">Full Time Employees</th>
+              <td>{company?.fullTimeEmployees}</td>
+            </tr>
+            <tr>
+              <th scope="col">Country</th>
+              <td>{company?.country}</td>
+            </tr>
+            <tr>
+              <th scope="col">City</th>
+              <td>{company?.city}</td>
+            </tr>
+            <tr>
+              <th scope="col">Phone</th>
+              <td>{company?.phone}</td>
+            </tr>
+            <tr>
+              <th scope="col">Address</th>
+              <td>{company?.address}</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  );
+};
```

Comment:
- [ Optional ] I think it would be better if your `details page` followed the design provided on Behance.net, as a web  developer following design templates is a crtical skill that you would want the world to know you have. Please observe the differences in the screenshots below and style the details page according to it.
   
   - The background colors are missing from your design 
   - The zebra stripped design is also missing from your design

## Project wireframe
![Screenshot from 2023-04-22 18-27-45](https://user-images.githubusercontent.com/16392046/233793445-14217dce-d119-4839-b941-4f16f33793eb.png)

## Your design
![Screenshot from 2023-04-22 18-33-58](https://user-images.githubusercontent.com/16392046/233793475-7a6b9b16-593c-4688-824c-9384ea3edc4f.png)